mule-extensions-api/src/main/java/org/mule/runtime/extension/api/declaration/type/annotation/CustomDefinedStaticTypeAnnotation.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.extension.api.declaration.type.annotation;
+
+import org.mule.metadata.api.annotation.MarkerAnnotation;
+import org.mule.metadata.api.annotation.TypeAnnotation;
+import org.mule.metadata.api.model.ObjectType;
+
+/**
+ *
+ * @since 1.1
+ */
+public class CustomDefinedStaticTypeAnnotation extends MarkerAnnotation {
```

Comment:
if this is not public, then why is in here? can't  this be moved to extension-support?

Reply:
is the same case as InfrastructureTypeAnnotation, we could move them both to extensions-support but maybe there was a good reason to put it there.

Reply:
today we dont have any type annotation outside mule-extensions-api,